ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Extension.java

```diff
@@ -115,6 +115,6 @@ public void initializeParsers(ExtensionParsingContext context) {
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_1_5, EJB3Subsystem15Parser.INSTANCE);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_2_0, EJB3Subsystem20Parser.INSTANCE);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_3_0, EJB3Subsystem30Parser.INSTANCE);
-        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_3_1, EJB3Subsystem31Parser.INSTANCE);
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_4_0, EJB3Subsystem40Parser.INSTANCE);
```

Comment:
looking at when 3.1 was introduced, it looks like it was a mistake it was even called 3.1 as it was introduced in 10.x development and merged only few weeks back. 
It should already be 4.0.
can you please remove 3.1 and just have 4.0 with all current changes in it.